### ISSUE TITLE
"@user is typing" message does not disappear in the thread window

### ISSUE THREAD

**Issue by marvinwankersteen** [matterhorn-chat/matterhorn#777]
Hey,

I noticed a bug in thread mode.
The `[@user is typing]`-message does not disappear in the thread window. It stays until the thread window is closed and reopened.
In the main window, however, the correct `[@user is typing]`-status is visible when the thread window is open.

```
matterhorn 50200.17.0 (master@7ca52c1)
 using mattermost-api 50200.13.0 (HEAD@273adac)
```

**Comment by jtdaugherty** [comment]
Thanks for reporting this. I'll investigate.

**Comment by jtdaugherty** [comment]
I looked into this briefly and wanted to make a note here for when I have a bit of time for a fix: the issue is that `startTypingUsersRefreshThread` simply does not set up expirations for the typing data for threads. It only iterates over channels. It should be straightforward to extend it to threads.

**Comment by jtdaugherty** [comment]
This is now fixed in 6f80e6827ea349d7581d880cae3b1c2038a2104a on `develop`. Thanks again!
